main.py

```
@@ -0,0 +1,40 @@
+
+@app.get("/")
+def read_root():
+    return {"message": "클라우드 커뮤니티 API 서버가 실행 중입니다!"}
+
+@app.get("/health")
+def health_check():
+    return {"status": "healthy"}
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run("main:app", host="0.0.0.0", port=8001, reload=True)
```

Ruff (0.14.11)

Possible binding to all interfaces

(S104)